tests/integrations/realcluster/cluster.go

```diff
@@ -165,3 +182,58 @@ func waitTiupReady(t *testing.T, tag string) {
 	}
 	require.Failf(t, "TiUP is not ready", "tag: %s", tag)
 }
+
+type msClusterSuite struct {
```

Comment:
seems we can add a field `ms bool` to `clusterSuite` rather than create a new structure.

Reply:
good point